src/main/java/com/github/rodionovsasha/commentservice/repositories/TopicRepository.java

```diff
@@ -0,0 +1,19 @@
+package com.github.rodionovsasha.commentservice.repositories;
+
+import com.github.rodionovsasha.commentservice.entities.Topic;
+import org.springframework.data.domain.Pageable;
+import org.springframework.data.domain.Sort;
+import org.springframework.stereotype.Repository;
+
+import java.util.List;
+import java.util.Optional;
+
+@Repository
+public interface TopicRepository extends BaseRepository<Topic> {
+    @Override
+    Optional<Topic> findOne(long id);
```

Comment:
Do you really need to override this method explicitly in every repository given that it does exist in `BaseRepository`?

Reply:
not really